org.hydrogenmusic.hydrogen.json

```diff
@@ -0,0 +1,174 @@
+{
+  "app-id": "org.hydrogenmusic.hydrogen",
+  "branch": "stable",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
```

Comment:
Have you tried 5.12?

Reply:
jack1 fails to build on this 5.12.

Reply:
How fails exactly? Is it something that we can patch here or it's completely broken?

Reply:
it is missing db.h it seems.

Reply:
https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/608

Reply:
It would be better to just embed it in the application and switch to 5.12.

Reply:
Apparently it wasn't meant to be removed. And libdb doesn't seem to be easy (a tonne of patches amongst other things)

Reply:
WiP to be fixed: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/merge_requests/929

Reply:
Bumped jack to 1.9.12 and lash seems to be api with it. SDK 5.12 now.